### ISSUE TITLE
Fix use of deprecated boost::bind global namespace placeholders

### ISSUE THREAD

**Issue by robtaylor** [lip6/coriolis#37]
```
In file included from /usr/include/boost/bind.hpp:30,
                 from ../../hurricane/src/viewer/CellPrinter.cpp:20:
/usr/include/boost/bind.hpp:36:1: note: ‘#pragma message: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior.’
   36 | BOOST_PRAGMA_MESSAGE(
      | ^~~~~~~~~~~~~~~~~~~~
[426/763] Compiling C++ object hurrica.../viewer/libviewer.so.p/CellImage.cpp.o
In file included from /usr/include/boost/bind.hpp:30,
                 from ../../hurricane/src/viewer/CellImage.cpp:21:
/usr/include/boost/bind.hpp:36:1: note: ‘#pragma message: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior.’
   36 | BOOST_PRAGMA_MESSAGE(
      | ^~~~~~~~~~~~~~~~~~~~
```

**Comment by jpc-lip6** [comment]
Fixed in #2633fb54 from ``wip-gf180mcu``. With additional drop of Qt 4 deprecated functions.

**Comment by robtaylor** [comment]
Can you make a pull request fixing just this issue against this issue?
